### ISSUE TITLE
Evaluation protocols on SimplerEnv

### ISSUE THREAD

**Issue by ygtxr1997** [intuitive-robots/MoDE_Diffusion_Policy#3]
![image](https://github.com/user-attachments/assets/db9c18f9-a0be-451c-ae41-17ddeed91604)

As your paper said, you used 6 subsets of Open-X Embodiments dataset to pretrain the MoDE and further **finetuned** the model.
But I cannot find more details about this **finetuning** stage in your paper. Could you please give more implementation details about the experiments on SimplerEnv? 
I would be very appreciate if you can share the finetuning data, training hyper-parameters, evaluation settings, and even the code!

**Comment by mbreuss** [comment]
Hi,

For running MoDE experiments we just used the pretrained checkpoint available at: https://huggingface.co/mbreuss/MoDE_Pretrained
The sentence with finetuning refers to this Repo when training on CALVIn and LIBERO. For the SIMPLER results we did not additional training after the initial pretraining phase. So the checkpoint is usable for SIMPLER.

You can view full training configs and run logs at: https://api.wandb.ai/links/irl-masterthesis/ql9m7m5i

I dont have a public clean version for pretraining and SIMPLER code yet but I will publish a new repo soon that will also contain MoDe for pretraining and SIMPLER evals after the RSS deadline. Sorry for that delay. 

Let me know if you have any other questions.

**Comment by mbreuss** [comment]
I padded images with some zero representative tokens. Right now the code for that is not public, as its part of another pretraining repo but I will try to release it next month. I just checked the normalized values and they do not match the bridge variants I used. Is it urgent for the evals then send me an email and I will share some code with you.

**Comment by ygtxr1997** [comment]
Hi @mbreuss, I have sent you an email to your email address. Have you received it?
